firestarter/common/validations.py

```diff
@@ -0,0 +1,27 @@
+from jsonschema import validate
+import os
+import json
+import yaml
+
+
+def helper_get_task_manager_schema(schema_path: str) -> dict:
+    TASK_MANAGER_SCHEMA = False
+
+    with open(os.path.join(os.path.dirname(__file__), schema_path)) as schema:
+        TASK_MANAGER_SCHEMA = json.load(schema)
+
+    return TASK_MANAGER_SCHEMA
+
+
+def validate_task_manager(task_manager_path: str, schema_path: str) -> dict:
+    with open(task_manager_path, 'r') as task_manager:
+        task_manager_data: dict = yaml.load(
```

Comment:
Should not we use `safe_load`?

According to [docs](https://pyyaml.org/wiki/PyYAMLDocumentation)
> Warning: It is not safe to call yaml.load with any data received from an untrusted source! yaml.load is as powerful as pickle.load and so may call any Python function.

Reply:
Updated!